eng/pipelines/templates/jobs/mgmt-release.yml

```diff
@@ -39,21 +39,13 @@ jobs:
 
       - ${{ if and(eq(parameters.ShouldPublishToNuget, 'true'), eq(variables['Build.Repository.Name'], 'Azure/azure-sdk-for-net')) }}:
 
-        - task: Powershell@2
-          displayName: 'Verify Package Tags and Create Git Releases'
-          inputs:
-            filePath: ${{ parameters.BuildToolsPath }}/scripts/create-tags-and-git-release.ps1
-            arguments: >
-              -artifactLocation '$(PIPELINE.WORKSPACE)/packages'
-              -workingDirectory $(System.DefaultWorkingDirectory)
-              -packageRepository 'Nuget'
-              -releaseSha '$(Build.SourceVersion)'
-              -repoOwner 'Azure'
-              -repoName 'azure-sdk-for-net'
-            pwsh: true
-          env:
-            GH_TOKEN: $(azuresdk-github-pat)
-          condition: and(succeeded(), ne(variables['SkipCreateTagAndGitRelease'], 'true'))
+        - template: /eng/common/pipelines/templates/steps/create-tags-and-git-release.yml
+          parameters:
+            ArtifactLocation: '$(PIPELINE.WORKSPACE)/packages'
+            WorkingDirectory: $(System.DefaultWorkingDirectory)
```

Comment:
WorkingDirectory doesn't need to be set since we're no longer running a script rooted in the BuildToolsPath. The [regular usage of this in the net release yml](https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/templates/stages/archetype-net-release.yml#L108) doesn't set it either.

Reply:
I updated it to remove working directory